### PR TITLE
(Publish TBD) Whats new: lookout visualization update

### PR DESCRIPTION
**Pull request by hollorandan**
Please follow [conventional commit standards](https://www.conventionalcommits.org/en/v1.0.0/)
in your commit messages and pull request title.

## Give us some context

* This should be published sometime in the morning of Tuesday, April 18th.
* This does not require PMM review.

### CODE REVIEW

File: src/content/whats-new/2023/04/whats-new-04-06-alert-condition-form.md [hunk 1]
```diff
@@ -1,8 +1,10 @@
 ---
 title: 'Streamline alert configuration and management'
-summary: 'Set up alerts faster and with more flexible configuration options.'
+summary: 'Set up alerts faster and with more flexible configuration options.’
+
 releaseDate: '2023-04-06'
```
Comment (nateborr) [inline src/content/whats-new/2023/04/whats-new-04-06-alert-condition-form.md:5]: Can a PR reviewer change this release date to `'2023-04-19'` for us? Or would we need to fork his repo and push up a new PR?

Comment (rhetoric101) [reply src/content/whats-new/2023/04/whats-new-04-06-alert-condition-form.md:5]: @nateborr I can change it for you.

Comment (rhetoric101) [reply src/content/whats-new/2023/04/whats-new-04-06-alert-condition-form.md:5]: @nateborr I can change it for you.

**Comment by rhetoric101** [comment]
I had trouble getting this to build. I'll follow up next week!

**Comment by nateborr** [comment]
@rhetoric101 Did we lose the content from this update? I'm not seeing the Lookout related text anymore.

**Comment by rhetoric101** [comment]
I am closing this because this PR was abandoned by its creator. It has been replaced by this PR: 12686

**Pull request closed without merging**